test/com/gu/contentapi/sanity/support/FakeAppSupport.scala

```diff
@@ -1,19 +1,12 @@
 package com.gu.contentapi.sanity.support
 
-import org.scalatest.{TestData, Suite}
-import org.scalatestplus.play.{OneAppPerTest, OneAppPerSuite}
-import play.api.GlobalSettings
-import play.api.test.FakeApplication
+import org.scalatest.{TestData, TestSuite}
+import org.scalatestplus.play.{OneAppPerTest, OneAppPerSuite, FakeApplicationFactory}
 
-trait FakeAppSupport extends OneAppPerSuite { self: Suite =>
+trait FakeAppSupport extends OneAppPerSuite 
+  with FakeApplicationFactory { self: TestSuite =>
 
-  override implicit lazy val app = new FakeApplication(
-
-    // override Global so that we don't start the Quartz scheduler
-    withGlobal = Some(new GlobalSettings {}),
-
-    // don't trip up on the Guardian self-signed certificate in CODE
-    additionalConfiguration = Map("ws.acceptAnyCertificate" -> true))
```

Comment:
We recently added a new certificate on `CODE` but you should double checked it works fine on CODE before merging

Reply:
Yep, will do now 👍